tests/infra_test.go

```diff
@@ -996,6 +999,255 @@ var _ = Describe("[Serial]Infrastructure", func() {
 		})
 
 	})
+
+	Describe("Node-labeller", func() {
+		Context("basic labelling", func() {
+			var nodesWithKVM []*k8sv1.Node
+
+			BeforeEach(func() {
+				tests.BeforeTestCleanup()
+				nodesWithKVM = tests.GetNodesWithKVM()
+			})
+
+			It("label nodes with cpu model and cpu features", func() {
+				for _, node := range nodesWithKVM {
+					node, err := virtClient.CoreV1().Nodes().Get(context.Background(), node.Name, metav1.GetOptions{})
```

Comment:
The `nodesWithKVM` array contains node objects returned by a `Get()` call. Is it needed to call `Get()` again here?